yal/utils.py

```diff
@@ -76,11 +77,22 @@ def normalise_phonenumber(phonenumber):
         raise ValueError("Invalid phone number")
 
 
+def extract_first_emoji(persona_emoji):
+    if is_emoji(persona_emoji):
+        return persona_emoji
```

Comment:
Do we still need this `is_emoji` check? Could we not just rely on the `emoji_list` function completely?